src/Button/Button.react.js

```diff
@@ -56,15 +56,15 @@ const contextTypes = {
 };
 
 function getStyles(props, context, state) {
-    const { button, buttonFlat, buttonRaised, buttonDisabled, buttonRaisedDisabled } = context.uiTheme;
+    const { button, buttonFlat, buttonRaised } = context.uiTheme;
+    const { palette, buttonDisabled, buttonRaisedDisabled } = context.uiTheme;
```

Comment:
I don't have this setup on my computer anymore (new job = new computer) plus my repo is now 10 commits behind... could you look into this?  I just got put on project and probably wont' have time.